test/main_test.go

```diff
@@ -281,3 +290,76 @@ func deleteClusterWideTestOperators() {
 		}
 	}
 }
+
+func monitorOlmNamespace() {
+	monitorNamespace(config.GetOlmNamespace())
+}
+
+func monitorKogitoOperatorNamespace() {
+	monitorNamespace(kogitoClusterWideNamespace)
+}
+
+func monitorNamespace(namespace string) {
```

Comment:
shouldn't we have that one in framework package ?

Reply:
Will directly call `StartPodLogCollector`